src/Packagist/WebBundle/Resources/views/Package/suggesters.html.twig

```diff
@@ -0,0 +1,29 @@
+{% extends "PackagistWebBundle::layout.html.twig" %}
+
+{% set showSearchDesc = 'hide' %}
+
+{% block head_additions %}<meta name="robots" content="noindex, nofollow">{% endblock %}
+
+{% block title %}Suggesters Packages - {{ name }} - {{ parent() }}{% endblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-xs-12 package">
+            <div class="package-header">
+                <h2 class="title">
+                    <a href="{{ path("view_package", {name: name}) }}">{{ name }}</a> suggesters
+                    <small>({{ count }})</small>
+                </h2>
+            </div>
+        </div>
+    </div>
+
+    <section class="row">
+        <section class="col-md-12">
```

Comment:
@Soullivaneuh this is not an issue. This is what a row does in bootstrap:

- clearfix to ensure floated columns are kept inside it
- negative left and right margins to compensate the column padding

and here is what a column does:

- float left
- set size (``col-*-12`` is full width)
- adding padding to create the gutter

So when using a row with a single -12 column in it, you just create a block talking all the width of the parent (but using 2 divs to achieve it). And this is exactly the behavior of a block element in HTML when no additional styles are applied.

there is nothing in bootstrap forcing to have everything nested inside rows (this is needed only when using a multi-column layout).